### DISCUSSION TITLE
Passing reportOptions

### DISCUSSION THREAD

**Discussion by irecevic** [saleweaver/python-amazon-sp-api#625] [Q&A] [answered]
When creating the report GET_VENDOR_SALES_REPORT, we deliver all details for the report creation

ReportsV2(marketplace=Marketplaces.xx, credentials=credentials).create_report(
        reportType='GET_VENDOR_SALES_REPORT',
        dataStartTime='2022-01-01T00:00:00.000001',
        dataEndTime=(datetime.utcnow() - timedelta(days=9)).isoformat(),
        marketplaceIds=['A1F83G8C2ARO7P'],
        reportOptions= [{
            **"reportPeriod":"MONTH",
            "distributorView": "SOURCING",
            "sellingProgram":"RETAIL"**}])

When checking the report document, there is an error.

 'payload': {'compressionAlgorithm': 'GZIP',
             'document': '{\n'
                         '  "errorDetails" : "This report type requires the '
                         'reportPeriod, distributorView, sellingProgram '
                         'reportOption to be specified. Please review the '
                         'document for this report type on GitHub, provide a '
                         'value for this reportOption in your request, and try '
                         'again."\n'
                         '}'

**To Reproduce**

Use above code.

**Expected behavior**

Return of the report document. Instead there is an error even if the reportOptions are passed to **create_report**

**Desktop (please complete the following information):**

 - OS: MacOS

**Comment by saleweaver** [comment, marked as answer]
reportOptions never is an array.

**Reply by irecevic** [reply]
Thank you @saleweaver .
But how to pass the parameters required for GET_VENDOR_SALES_REPORT? The [documentation](https://developer-docs.amazon.com/sp-api/docs/report-type-values#vendor-retail-analytics-reports) states array like value for **reportOptions**.
